lib/browser/results.js

```diff
@@ -18,13 +18,11 @@
 
 'use strict';
 
+import Collection, { createCollection } from './collections';
 import { objectTypes } from './constants';
-import { createCollection, createMethods } from './util';
+import { createMethods } from './util';
 
-export class Results {
-    constructor() {
-        throw new TypeError('Illegal constructor');
-    }
+export default class Results extends Collection {
```

Comment:
So you can do `import Results from 'results'` rather than `import {Results} from 'results'`. It just the canonical way of exporting the primary object of a module.